### PR TITLE
Fix broken links in resources/index

### PR DESCRIPTION
**Pull request by mateusppereira**
Links removed doesn't exists or cannot be accessed anymore

### CODE REVIEW

File: src/pages/resources/index.mdx [hunk 1]
```diff
@@ -21,20 +21,16 @@ These resources have been gathered to help new users learn about OpenLineage, or
 ## Conference Talks
 
 * [OpenLineage in Airflow: A Comprehensive Guide](https://www.youtube.com/watch?v=SZBVgREqets) - Maciej Obuchowski @ Airflow Summit (2023)
-* [Cross-platform Data Lineage with OpenLineage](https://www.databricks.com/dataaisummit/session/cross-platform-data-lineage-openlineage/) - Julien Le Dem and Willy Lulciuc @ Data+AI Summit (2023)
```
Comment (pawel-big-lebowski) [inline src/pages/resources/index.mdx:24]: isn't this available here https://www.youtube.com/watch?v=rO3BPqUtWrI&t=2s ?

File: src/pages/resources/index.mdx [hunk 2]
```diff
@@ -21,20 +21,16 @@ These resources have been gathered to help new users learn about OpenLineage, or
...
 * [Column-level Lineage is Coming to the Rescue](https://youtu.be/xFVSZCCbZlY) - Paweł Leszczyński and Maciej Obuchowski @ Berlin Buzzwords (2023)
 * [Cross-platform Data Lineage with OpenLineage](https://www.youtube.com/watch?v=pLBVGIPuwEo) - Julien Le Dem @ Berlin Buzzwords (2022)
 * [Data Observability and Pipelines: OpenLineage and Marquez](https://mattturck.com/datakin/) - Matt Turck @ Data Driven NYC (2021)
 * [Data Lineage and Observability with Marquez and OpenLineage](https://bigdatatechwarsaw.eu/edition-2021/) - Julien Le Dem @ Big Data Technology Warsaw Summit (2021)
 * [Data Lineage with Apache Airflow using OpenLineage](https://airflowsummit.org/sessions/2021/data-lineage-with-apache-airflow-using-openlineage/) - Julien Le Dem & Willy Lulciuc @ Airflow Summit (2021)
-* [Observability for data pipelines with OpenLineage](https://2021.berlinbuzzwords.de/session/observability-data-pipelines-openlineage) - Julien Le Dem @ Berlin Buzzwords (2021)
```
Comment (pawel-big-lebowski) [inline src/pages/resources/index.mdx:30]: https://www.youtube.com/watch?v=HEJFCQLwdtk

**Review by pawel-big-lebowski — COMMENTED** [review]
Thanks for the contribution.
Some of the resources are available on Youtube. 
Would you mind updating the link instead of removing?

**Comment by pawel-big-lebowski** [comment]
@mateusppereira any plans or accomplishing it or shall I take my comments into separate PR and close this one?

**Comment by mateusppereira** [comment]
@pawel-big-lebowski just fixed

**Comment by pawel-big-lebowski** [comment]
@mateusppereira great, could you please sign off your commit? Clicking the failing DCO step will provide you with detailed instruction.

**Comment by mateusppereira** [comment]
@pawel-big-lebowski ✅

**Review by pawel-big-lebowski — APPROVED** [review]

**Pull request merged**